src/guidescanpy/flask/templates/job_query.html

```diff
@@ -156,32 +137,69 @@ <h5 class="card-header">{{ region }}</h5>
                         };
                     igv.createBrowser(igvDiv, options);
 
+                    var region = $('.resulttable').data('region');
                     var table = $('.resulttable').DataTable({
```

Comment:
This will need to go in a loop for each datatable found (looking for class `.resulttable` is okay).